packages/core/src/codecatalyst/commands.ts

```diff
@@ -222,6 +230,9 @@ export class CodeCatalystCommands {
     }
 
     public createDevEnv(): Promise<void> {
+        if (vscode.env.remoteName === 'ssh-remote') {
```

Comment:
will this prevent users from creating dev environments from remote machines that aren't dev environments?